docs/reference/commandline/service_create.md

```diff
@@ -761,6 +762,18 @@ $ docker service create --name dns-cache -p 53:53/tcp -p 53:53/udp dns-cache
 $ docker service create --name dns-cache -p 53:53/udp dns-cache
 ```
 
+### Provide credential specs for managed service accounts (Windows only)
+
+This option is only used for services using Windows containers. The
+`--credential-spec` must be in the format `file://spec.txt` or
+`registry://keyname`.
+
+When using the `file://` format, the referenced file must be present on the
+daemon host in the `C:\programfiles\docker\credentialspecs\` directory. When
```

Comment:
Let's make clear whether the path after `file://` should be a full path or a relative path within `C:\programfiles\docker\credentialspecs\`.

As a side note, it's potentially confusing if `file://` is meant to be used with a relative path here.